codegen/generators/php.rb

```diff
@@ -53,11 +53,11 @@ def nullable?(property_name, schema)
     end
 
     def scalar?(property)
-      property.key?('type') && language_translations_for_data_types.key?(property['type'])
+      property.key?('type') && select_language_translations_for_data_types(property['type'], property)
```

Comment:
This will return `false` or a non-boolean value which I haven't checked downstream if this matters, but it could